src/kw/arg.hpp

```diff
@@ -16,7 +16,7 @@ namespace kw
     struct arg
     {
         arg()
-        { }
+        {}
```

Comment:
guideline for this, I usually write ``{ }``.
It's also in the coding style examples. http://docs.steinwurf.com/coding_style.html